package.json

```
@@ -60,18 +60,18 @@
         "@bull-board/koa": "^3.7.2",
         "@emotion/react": "11.7.1",
         "@emotion/styled": "11.6.0",
-        "@ezs/analytics": "^1.18.4",
-        "@ezs/basics": "^1.17.1",
+        "@ezs/analytics": "^2.0.1",
+        "@ezs/basics": "^1.22.3",
         "@ezs/booster": "^1.2.4",
-        "@ezs/conditor": "^2.5.0",
-        "@ezs/core": "1.28.2",
-        "@ezs/istex": "1.5.1",
-        "@ezs/lodex": "1.15.0",
-        "@ezs/loterre": "^1.3.2",
-        "@ezs/sparql": "^1.2.0",
-        "@ezs/storage": "^1.6.2",
+        "@ezs/conditor": "^2.8.2",
+        "@ezs/core": "2.0.1",
+        "@ezs/istex": "1.5.3",
+        "@ezs/lodex": "1.15.2",
+        "@ezs/loterre": "^2.0.0",
+        "@ezs/sparql": "^1.2.1",
+        "@ezs/storage": "^1.7.0",
         "@ezs/transformers": "1.3.11",
```

```suggestion
        "@ezs/transformers": "1.4.1",
```

Why not update to the last version of `@ezs/transformers` ?

---

Because of the change related to URI generation. A patch is need to update...